### PR TITLE
[Core] Add CalculateOnIntegrationPoints and SetValuesOnIntegrationPoints for int-type

### PR DESCRIPTION
**Pull request by tteschemacher**
Hi all,

I am completing the interface for the Calculate and SetValues functions.

### CODE REVIEW

**Comment by philbucher** [comment]
Can you please add a template for the Datatype? (CalculateOn...)
Then you don’t have to copy paste the fct for each datatype

Not sure id this also works for SetValue...

**Comment by tteschemacher** [comment]
sure good idea!

**Comment by VeronikaSinger** [comment]
Thanks Tobi. Works fine now for my problem case.

File: kratos/python/add_mesh_to_python.cpp [hunk 1]
```diff
@@ -146,60 +146,20 @@ py::list GetIntegrationPointsFromElement( Element& dummy )
...
+pybind11::list CalculateOnIntegrationPoints(
+    TObject& dummy, const Variable<TDataType>& rVariable, const ProcessInfo& rProcessInfo)
 {
-    std::vector<Vector> Output;
-    dummy.CalculateOnIntegrationPoints( rVariable, Output, rProcessInfo);
+    std::vector<TDataType> Output;
+    dummy.CalculateOnIntegrationPoints(rVariable, Output, rProcessInfo);
     pybind11::list result;
-    for( unsigned int j=0; j<Output.size(); j++ )
+    for (unsigned int j = 0; j < Output.size(); j++)
```
Comment (philbucher) [inline kratos/python/add_mesh_to_python.cpp:156]: I like the other style more :)

**Review by philbucher — APPROVED** [review]
thx!

**Comment by tteschemacher** [comment]
thanks @VeronikaSinger for fixing

**Pull request merged**